x/inflation/types/v1beta2/params.go

```diff
@@ -0,0 +1,95 @@
+package v1beta2
+
+import (
+	"strconv"
+
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"github.com/pkg/errors"
+)
+
+var _ paramtypes.ParamSet = (*Params)(nil)
+
+const (
+	DefaultInflationDecayFactor uint32 = 2 // years
+	DefaultInitialInflation     string = "100.0"
+	DefaultVarince              string = "0.05"
```

Comment:
Make these `sdk.Decimal` or even integers that are used to populate a decimal later.  Maybe look at how some other parameters in cosmos-sdk are implemented.

Reply:
Done